src/main/perl/TextRender.pm

```diff
@@ -154,16 +153,10 @@ Java properties format (using C<Config::Properties>),
 
 .INI format (using C<Config::Tiny>)
 
-=item general
-
-(using C<Config::General>).
-
-Caution: there is no way to garantee reproducible results as
-there is no control over the order of hash keys. It is recommended to
-try to generate this format in another way (e.g. via TT).
-
 =back
 
+(Builtin module <general> is provided as part of L<CCM::TextRender> since 15.10).
```

Comment:
`general` is not available in `CAF::TextRender`, only in `CCM::TextRender`, which is a subclass of `CAF::TextRender`.  i don't mind readding the item, but i think this is confusing as e.g. following code will not work
```perl
use CAF::TextRender;

my $trd = CAF::TextRender->new('general', {some => 'hashref'});
# do something useful with $trd
```